Documentation/architecture.rst

```diff
@@ -199,243 +235,792 @@ Example
+  implemented:
+
+  * **Allow/Requires:** Connectivity policy, e.g. allow a pod to talk to
+    another pod
+  * **L4** L4 connectivity policy
+
+**Example:**
+
+The following example describes a rule which applies to all endpoints which
+carry the label `id.service1`. 
+
+::
+
+	[{
+		"coverage": ["role=backend"],
+		"allow": allowData
+	}]
+
+Allow Rules
+-----------
+
+This is the simplest rule type. The rule defines a list of labels which are
+allowed to consume whatever endpoints are covered by the coverage.
+
+If an endpoint transmits to another endpoint and the communication is not
+permitted by at least one *allow* rule, all packets of the connection will be
+silently dropped.
+
+
++---------------+----------+---------------------------------------------------+
+| Field         | Type     | Description                                       |
++---------------+----------+---------------------------------------------------+
+| coverage      | Array of | List of labels which must match in order for this |
+|               | labels   | rule to be applied.                               |
++---------------+----------+---------------------------------------------------+
+| allow         | Array of | List of labels which are allowed to initiate a    |
+|               | allows   | connection to any endpoint covered by coverage.   |
++---------------+----------+---------------------------------------------------+
+
+allow:
+
++---------------+----------+---------------------------------------------------+
+| Field         | Type     | Description                                       |
++---------------+----------+---------------------------------------------------+
+| action        | string   | { "accept", "always-accept", "deny" }             |
++---------------+----------+---------------------------------------------------+
+| label         | label    | Allowed or denied label
++---------------+----------+---------------------------------------------------+
+
+A short form is available as alternative to the above verbose JSON syntax:
+
++---------------+----------+---------------------------------------------------+
+| Field         | Type     | Description                                       |
++---------------+----------+---------------------------------------------------+
+| coverage      | Array of | List of labels which must match in order for this |
+|               | strings  | rule to be applied.                               |
++---------------+----------+---------------------------------------------------+
+| allow         | Array of | List of labels which are allowed to initiate a    |
+|               | strings  | connection to any endpoint covered by coverage.   |
+|               |          | The action is "accept" unless the label has the   |
+|               |          | prefix `!` in which case the action is "deny".    |
++---------------+----------+---------------------------------------------------+
```

Comment:
In the `readthedocs`, somehow this rightmost cell is cut off to fit the page, scrolling to the right by marking the text.